src/payment/strategies/masterpass/masterpass-payment-strategy.spec.ts

```diff
@@ -0,0 +1,265 @@
+import { createClient as createPaymentClient } from '@bigcommerce/bigpay-client';
+import { Action } from '@bigcommerce/data-store';
+import createAction from '@bigcommerce/data-store/lib/create-action';
+import { createRequestSender, RequestSender } from '@bigcommerce/request-sender';
+import { createScriptLoader } from '@bigcommerce/script-loader';
+import { Observable } from 'rxjs';
+
+import {
+    PaymentActionCreator, PaymentInitializeOptions,
+    PaymentMethod,
+    PaymentMethodActionCreator, PaymentMethodActionType,
+    PaymentMethodRequestSender, PaymentRequestSender
+} from '../../';
+import { getCartState } from '../../../cart/carts.mock';
+import { createCheckoutStore, CheckoutRequestSender, CheckoutStore, CheckoutValidator } from '../../../checkout';
+import { getCheckoutState } from '../../../checkout/checkouts.mock';
+import { MissingDataError } from '../../../common/error/errors';
+import { getConfigState } from '../../../config/configs.mock';
+import { getCustomerState } from '../../../customer/customers.mock';
+import { OrderActionCreator, OrderActionType, OrderRequestSender } from '../../../order';
+import { PaymentActionType } from '../../payment-actions';
+import { getMasterpass, getPaymentMethodsState, getStripe } from '../../payment-methods.mock';
+
+import { MasterpassPaymentStrategy, MasterpassScriptLoader } from './';
+import { Masterpass } from './masterpass';
+import { getMasterpassScriptMock } from './masterpass.mock';
+
+describe('MasterpassPaymentStragegy', () => {
+    // Described class
+    let strategy: MasterpassPaymentStrategy;
+
+    // Strategy's constructor dependencies
+    let orderRequestSender: OrderRequestSender;
+    let requestSender: RequestSender;
+
+    // Strategy's constructor arguments
```

Comment:
What's the difference between `strategy's constructor dependencies` and `strategy's constructor arguments`? Anyway,  these comments seem redundant, could you please remove unless you really want to keep them.

Reply:
that was a typo i will removed